### DISCUSSION TITLE
Welcome to AZ400-DesigningandImplementingMicrosoftDevOpsSolutions Discussions!

### DISCUSSION THREAD

**Discussion by LuizMacedo** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#272] [Announcements]
## 👋 Welcome!
  We've enabled discussions _experimentally_ in the AZ-400 repository and will monitor and evaluate how it goes.

  Discussions in the MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions repo should be focused on technical discussions relating to the Labs. It will be a place to help us with design decisions, discuss and hopefully resolve technical issues, make announcements, etc.

We hope that you:

- Ask questions you're wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we build together 💪.

We hope that you won't:

- Discuss issues not related to Labs.
- Support questions about Azure DevOps Demo Generator (those should still be directed to https://github.com/microsoft/AzureDevOpsDemoGenerator).
- Discuss or report issues for AZ-400 content on Microsoft Learn (those should still be directed to https://docs.microsoft.com/en-us/learn/support/faq?pivots=support or https://docs.microsoft.com/en-us/learn/support/troubleshooting#report-feedback).

**Comment by spilks** [comment]
Thank you, Luiz, this is a great forum and a way to discuss AZ-400! Nice job. -Kim
